### PR TITLE
fix: Fixed overlapping language case

### LINKED ISSUE #133: Put a check on overlapping languages( if one is contained in another)
## Description
As a developer,
I need to put a check on overlapping languages, like Java and Javascript

## Mocks
issue_bug

### Update [Required]
- [ ] Fix the bug

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Fixed issue of overlapping between Java and JavaScript

Fixes #133 

# Type of Change:
- Code
- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
answer_overlap_avoided

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: code/slack_functions.py [hunk 1]
```diff
@@ -369,7 +369,11 @@ def handle_message_answering(event_data):
         techs = techstack_vs_projects.keys()
         suggest_projects_set = set()
         for tech in techs:
-            if tech in text.upper():
+            search_text = text.upper()
```
Comment (sara-02) [inline code/slack_functions.py]: +1 to @ramitsawhney27 Maybe we can tokenize the text.upper() and do something like `set(tokens).intersectionset(langaguge_dict.key())`?

**Comment by sammy1997** [comment]
Updated with better and general solution @sara-02 @ramitsawhney27 !

**Comment by sammy1997** [comment]
Also checked the working. It's working fine

**Comment by sara-02** [comment]
@ramitsawhney27 You can review and merge this.

**Pull request merged**